.github/CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ Please send a pull request to the master branch. Please include [documentation](
 - Separate code commits from reformatting commits.
 - Provide tests for any newly added code.
 - Follow PEP8.
-- When committing only documentation changes please include [ci skip] in the commit message to avoid running tests on Travis-CI.
+- When committing only documentation changes please include [ci skip] in the commit message to avoid running tests on Travis-CI and AppVeyor.
```

Comment:
Not sure. Some docs are clearly small changes, but some aren't.  I'm not sure what the status is on failing the build if the doc build fails, but I know we out that in to prevent the docs being broken for long.